src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
-// write code here
+const list = document.querySelectorAll('li');
+
+function employee(info) {
```

Comment:
```suggestion
function getEmployee(info) {
```
функції називаємо дієсловами